Sources/KIF/Classes/KIFUIViewTestActor.m

```diff
@@ -411,7 +411,20 @@ - (void)activateCustomActionWithName:(NSString *)name expectedResult:(BOOL)expec
             return KIFTestStepResultFailure;
         }];
     }
+}
 
+- (void)performAccessibilityActivateWithExpectedResult:(BOOL)expectedResult;
+{
+    @autoreleasepool {
+        KIFUIObject *found = [self _predicateSearchWithRequiresMatch:YES mustBeTappable:YES];
+        
+        [self runBlock:^KIFTestStepResult(NSError **error) {
+            if([found.element accessibilityActivate] == expectedResult) {
+                return KIFTestStepResultSuccess;
```

Comment:
Should we wait for animations here before yielding back to the test? I'm presuming that `accessibilityActivate` is generally an approximation of a user action that might alter app state and cause transitions to happen?

Reply:
thats a good idea